code/modules/clothing/suits/armor.dm

```diff
@@ -84,24 +84,6 @@
 	armor = list(melee = 10, bullet = 10, laser = 80, energy = 50, bomb = 0, bio = 0, rad = 0)
 	siemens_coefficient = 0
 
-/obj/item/clothing/suit/armor/swat

```

Comment:
It's the former path to the snowflake armor deathsquid were using. I didn't notice the one underneath but it sounds even more snowflakey, I'm deleting that too.